coding-the-legislation/10_basic_example.md

```diff
@@ -9,25 +9,34 @@ class flat_tax_on_salary(Variable):
     column = FloatCol
     entity = Person
     label = u"Individualized and monthly paid tax on salaries"
+    definition_period = MONTH
 
     def function(person, period):
-        period = period.this_month
         salary = person('salary', period)
 
-        return period, salary * 0.25
+        return salary * 0.25
 ```
 
 Let's explain in details the different part of the code:
 - `class flat_tax_on_salary(Variable):` declares a new variable with the name `flat_tax_on_salary`.
 - Metadatas:
-  - `column = FloatCol` declares the type of the variable, here a Float. All available types are documented [here]().
+  - `column = FloatCol` declares the type of the variable. Possible types are :
+    - `BoolCol`: boolean
+    - `DateCol`: date
+    - `FloatCol`: float
+    - `IntCol`: integer
+    - `AgeCol`: age
+    - `FixedStrCol`: string with maximum length (deprecated)
+    - `EnumCol`: discrete value (from an enumerable)
```

Comment:
I thought they are never used so I deprecated them unilaterally, waiting for somebody to complain. I removed the deprecation.